velox/core/QueryConfig.h

```diff
@@ -188,6 +188,9 @@ class QueryConfig {
   static constexpr const char* kSpillableReservationGrowthPct =
       "spillable_reservation_growth_pct";
 
+  /// If true, array_agg() aggregation function will ignore nulls in the input.
+  static constexpr const char* kArrayAggIgnoreNulls = "array_agg_ignore_nulls";
```

Comment:
It would be nice to document this config in velox/docs/configs.rst

Would it make sense to add presto. prefix to the name to clarify that this config applies to Presto function? Similar to spark.xxx configs?

presto.array_agg.ignore_nulls

?